Cargo.toml

```diff
@@ -33,7 +33,7 @@ linkerd2-stack     = { path = "lib/stack" }
 linkerd2-task      = { path = "lib/task" }
 linkerd2-timeout   = { path = "lib/timeout" }
 
-linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", tag = "v0.1.4", version = "0.1.4" }
+linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", branch = "alex/retries" } #tag = "v0.1.4", version = "0.1.4" }
```

Comment:
This should probably be changed to pin the master SHA until a new release is cut. That is, unless @adleong wants to cut an 0.1.5 in the current state.